plugins.xml

```diff
@@ -12097,14 +12097,22 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<certification type="official"/>
 			<description>Datacite plugin  Bugfix release</description>
 		</release>
-		<release date="2024-06-19" version="3.4.0.1" md5="a83acc4e7a90fadd7c03c87e64931d8d">
-			<package>https://github.com/withanage/datacite/releases/download/v3_4_0-1/datacite-v3_4_0-1.tar.gz</package>
+		<release date="2025-10-13" version="3.4.0.2" md5="f2cc4984fc50d84ce5ecf14eb8868b5d">
+			<package>https://github.com/UB-Heidelberg/datacite/releases/download/v3_4_0-2/datacite-3_4_0-2.tar.gz</package>
```

Comment:
Hi @nongenti, it looks like there's an issue with this release package as it's empty when unzipped.